src/js/components/covid19/DateNote.jsx

```diff
@@ -4,15 +4,17 @@
  */
 
 import React from 'react';
-import PropTypes from 'prop-types';
+import { useSelector } from 'react-redux';
 
-const propTypes = { dateString: PropTypes.string };
 
-const DateNote = ({ dateString = 'June 30, 2020' }) => (
-    <div className="covid__date-note">
-        Data as of {dateString}
-    </div>
-);
+const DateNote = () => {
+    const date = useSelector((state) => state.covid19.latestSubmissionDate);
```

Comment:
I don't see how `DateNote` has access to redux state. Does using `useSelector` just somehow already have access to the store w/o using `mapStateToProps`?

Reply:
Also, are we actually using this component somewhere?

Reply:
yeah, we use it throughout the page.